cost/google/cloud_sql_idle_instance_recommendations/google_sql_idle_instance_recommendations.pt

```diff
@@ -341,52 +338,69 @@ script "js_sql_instances_cost_mapping", type:"javascript" do
+                    cur = ds_currency_reference[recommendation['currency']]['symbol']
                 } else {
-                    var cur = "$"
-                    var separator = ","
-                }
-                if (recommendation.costNanos < 0) {
-                    recommendation.costNanos = recommendation.costNanos * -1
-                }
-                if (recommendation.costUnits < 0) {
-                    recommendation.costUnits = recommendation.costUnits * -1
+                    cur = "$"
                 }
-                var strNano = "." + recommendation.costNanos
-                var combine = recommendation.costUnits + parseFloat(strNano)
-                var monthsDuration = parseFloat(recommendation.duration.replace("s", "")) / 2628288
-                var monthlySavings = combine / monthsDuration
-                total = total + monthlySavings
-                database['savings'] = (Math.round(monthlySavings * 1000) / 1000)
                 database['savingsCurrency'] = cur
-                database['priority'] = recommendation['priority']
-                database['primaryImpact'] = recommendation['primaryImpact']
-                database['recommenderSubtype'] = recommendation['recommenderSubtype']
-                database['state'] = recommendation['state']
-                database['region'] = recommendation['region']
-                database['service'] = "SQL"
-                var tags = []
-                if (database['tags'] != null) {
-                    Object.keys(database.tags).forEach(function (key) {
-                        tags.push(key + '=' + database.tags[key])
-                    });
-                }
-                database['tags'] = tags
-                instances.push(database)
+            } else {
+                cur = "$"
+                var separator = ","
             }
-        })
+
+            //Convert Costs from Negative to Positive and add Units and Nanos
+            if (!(recommendation.costNanos == null || recommendation.costNanos == undefined)) {
+              if (recommendation.costNanos != 0) {
+                recommendation.costNanos = recommendation.costNanos * -1
+              }
+            } else {
+              recommendation.costNanos = 0
+            }
+
+            if (recommendation.costUnits < 0) {
+                recommendation.costUnits = recommendation.costUnits * -1
+            }
+            var cost_nano = recommendation.costNanos * Math.pow(10, -9)
+            var combine = Number(recommendation.costUnits) + Number(parseFloat(cost_nano))
+            var monthsDuration = parseFloat(recommendation.duration.replace("s", "")) / 2628288
+            var monthlySavings = combine / monthsDuration
+            total = total + monthlySavings
+            database['savings'] = (Math.round(monthlySavings * 1000) / 1000)
+            database['savingsCurrency'] = cur
```

Comment:
I cannot prove this as we don't have any recommendations from testing, but it is likely that the policy incident will give an empty value for the 'description' column without the following:

```suggestion
            database['savingsCurrency'] = cur
            database['description'] = recommendation['description']
```

Reply:
Nvm, there isn't a description field in the policy incident. Might be worth adding that in

```ruby
      field "description" do
        label "Description"
      end
```